From 704868db61be1542c2d9e2b75ead00c45c56cc36 Mon Sep 17 00:00:00 2001
From: "Evgeniy A. Dushistov" <dushistov@mail.ru>
Date: Sat, 15 Aug 2020 15:09:08 +0300
Subject: fix build on Arch Linux
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

qtimageformats/src/plugins/imageformats/jp2/qjp2handler.cpp:844:41:
error: no declaration of «pow»
  844 |     const double jasperRate = minRate + pow((double(quality) / double(maxQuality)), 2) * maxRate;

Pick-to: 5.15
Change-Id: I085996c2db2251903b2a3e52e6e648831637c8f9
Reviewed-by: Shawn Rutledge <shawn.rutledge@qt.io>
[Retrieved from:
https://code.qt.io/cgit/qt/qtimageformats.git/commit/?id=704868db61be1542c2d9e2b75ead00c45c56cc36]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/plugins/imageformats/jp2/qjp2handler.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/imageformats/jp2/qjp2handler.cpp b/src/plugins/imageformats/jp2/qjp2handler.cpp
index 4311d26..05c7bc1 100644
--- a/src/plugins/imageformats/jp2/qjp2handler.cpp
+++ b/src/plugins/imageformats/jp2/qjp2handler.cpp
@@ -45,6 +45,7 @@
 #include "qcolor.h"
 
 #include <jasper/jasper.h>
+#include <math.h> // for pow
 
 QT_BEGIN_NAMESPACE
 
-- 
cgit v1.2.1

